src/change_stream/mod.rs

```diff
@@ -132,3 +155,36 @@ impl Stream for ChangeStream {
         todo!();
     }
 }
+
+// Specification used to create a new `ChangeStream`.
+#[derive(Debug)]
+pub(crate) struct ChangeStreamSpecification {
+    /// Specification used to create a new cursor.
```

Comment:
this docstring could be a little more specific, e.g. "...used to create the cursor wrapped by the `ChangeStream`."

Reply:
fixed